db/migrate/20170328035506_create_prices.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePrices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :prices do |t|
+      t.references :product, index: true
+      t.references :retailer, index: true
+      t.decimal :price
```

Comment:
Should we change the prices table to something like product_attributes or retailers_products which is a join table to keep price, rating, and so on?